Intersect.Editor/Forms/Editors/frmSpell.cs

```diff
@@ -1067,6 +1072,11 @@ private void txtSearch_Click(object sender, EventArgs e)
         }
 
         #endregion
+
+        private void cmbOverTimeAnimation_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            mEditorItem.OverTimeAnimation = AnimationBase.Get(AnimationBase.IdFromList(cmbOverTimeAnimation.SelectedIndex - 1));
```

Comment:
Put the animation id into a variable before calling Get, makes for easier debugging.

Reply:
Done, good call